client/network/src/protocol/sync.rs

```diff
@@ -448,7 +448,9 @@ impl<B: BlockT> ChainSync<B> {
 		block_announce_validator: Box<dyn BlockAnnounceValidator<B> + Send>,
 		max_parallel_downloads: u32,
 	) -> Self {
-		let mut required_block_attributes = BlockAttributes::HEADER | BlockAttributes::JUSTIFICATION;
+		let mut required_block_attributes = BlockAttributes::HEADER
+			| BlockAttributes::JUSTIFICATION
+			| BlockAttributes::JUSTIFICATIONS;
```

Comment:
Been testing this against older nodes and I'm getting an issue where the older nodes fail to decode `BlockAttributes` in a request with the extra bit set

Reply:
Switched this logic from `BlockAttributes` to the protobuf schema instead